cli/flox/doc/flox-activate.md

```diff
@@ -155,18 +155,29 @@ See [`manifest.toml(5)`](./manifest.toml.md) for more details on shell hooks.
 
 ## Variables used by `flox activate`
 
-`$FLOX_SHELL`
-:  When launching an interactive sub-shell, Flox launches the shell specified in
-   `$FLOX_SHELL` if it is set.
-   When printing a shell for sourcing in the current shell,
-   Flox will produce a script suitable for `$FLOX_SHELL` if it is set.
-
-`$SHELL`
-:  When launching an interactive sub-shell, Flox launches the shell specified in
-   `$SHELL` if it is set and `$FLOX_SHELL` is not set.
-   When printing a shell for sourcing in the current shell,
-   Flox will produce a script suitable for `$SHELL` if it is set
-   and `$FLOX_SHELL` is not set and Flox can't detect the parent shell.
+`$FLOX_SHELL`, `$FLOX_SHELL_NOCLOBBER`, `$SHELL`
```

Comment:
If we are going to document FLOX_SHELL_NOCLOBBER, I think the note below is sufficient
```suggestion
`$FLOX_SHELL`, `$SHELL`
```